src/Viz.js

```diff
@@ -98,6 +98,7 @@ export default class Viz extends BaseClass {
     this._groupBy = [accessor("id")];
     this._legend = true;
     this._legendConfig = {
+      ariaLabel: legendLabel.bind(this),
```

Comment:
looking at the chrome element inspector, it doesn't appear this is getting applied to the legend shapes. try moving this into the nested `shapeConfig` a few lines below, as `ariaLabel` is a property of the legend shapes, not the legend itself.